src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/utils.ts

```diff
@@ -123,3 +118,21 @@ export const areDefaultFiltersSelected = (
   if (!transportModes || transportModes.length === 0) return false;
   return transportModes.every((tm) => tm.selectedAsDefault === tm.selected);
 };
+
+const isVenue = (location: SearchLocation): boolean => {
+  return location.layer === 'venue';
+};
+
+const isGroupOfStopPlaces = (location: SearchLocation): boolean => {
+  return (
+    !!location.category?.length &&
+    location.category[0] === FeatureCategory.GROUP_OF_STOP_PLACES
+  );
+};
+
+export const getSearchPlace = (location: Location) =>
+  location.resultType === 'search' &&
+  (isVenue(location as SearchLocation) ||
+    isGroupOfStopPlaces(location as SearchLocation))
```

Comment:
We usually want to avoid using `as`.
What you can do instead, is let `isVenue` and `isGroupOfStopPlaces` also accept `Location`, and then check if the location has the property you're looking for.

```ts
const isVenue = (location: Location): boolean => {
  return 'layer' in location && location.layer === 'venue';
};
```
And then the same for `isGroupOfStopPlaces`, but looking for `'category'` instead of `'layer'`

Reply:
Nice to know. I´ll update the code👍